src/msg/async/AsyncConnection.cc

```diff
@@ -868,7 +868,7 @@ ssize_t AsyncConnection::_process_connection()
         if (r < 0)
           goto fail;
 
-        center->create_file_event(cs.fd(), EVENT_READABLE, read_handler);
+        center->create_file_event(cs.fd(), EVENT_WRITABLE, read_handler);
```

Comment:
hmm, if posix backend, this change is correct to remove extra line. But now async messenger supports multi backend which isn't fully follow posix semantic. By default we use readable event to indicate whether connection built and use writable event to indicate the extra posix semantic. So the original form is suitable..